<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inputs</class>
 <widget class="QMainWindow" name="inputs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(255, 122, 56)
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="player_type_label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>OCR A Extended</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 239, 60)</string>
    </property>
    <property name="text">
     <string>  Enter Player type</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QListWidget" name="player_type_list">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>160</y>
      <width>256</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 239, 60)</string>
    </property>
    <item>
     <property name="text">
      <string>Server</string>
     </property>
     <property name="font">
      <font>
       <family>OCR A Extended</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Client</string>
     </property>
     <property name="font">
      <font>
       <family>OCR A Extended</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="num_players_label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>270</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>OCR A Extended</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 239, 60)</string>
    </property>
    <property name="text">
     <string>Enter Number of Players</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QListWidget" name="num_players_list">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>250</y>
      <width>256</width>
      <height>101</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 239, 60)</string>
    </property>
    <item>
     <property name="text">
      <string>2</string>
     </property>
     <property name="font">
      <font>
       <family>OCR A Extended</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
     <property name="font">
      <font>
       <family>OCR A Extended</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
     <property name="font">
      <font>
       <family>OCR A Extended</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
     <property name="font">
      <font>
       <family>OCR A Extended</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
     <property name="font">
      <font>
       <family>OCR A Extended</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="input_title">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>OCR A Extended</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 239, 60)</string>
    </property>
    <property name="text">
     <string>Enter Game Player Inputs</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="inputs_button">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>500</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>OCR A Extended</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 239, 60)</string>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QLabel" name="game_option_label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>380</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>OCR A Extended</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 239, 60)</string>
    </property>
    <property name="text">
     <string>Game Options</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="check_box_frame">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>390</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 239, 60)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QCheckBox" name="color_drop_card_checkbox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>OCR A Extended</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Color-Drop Card</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
